src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutesSpec.scala

```diff
@@ -15,9 +15,7 @@ import scala.concurrent.ExecutionContext
 
 class LeoRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest {
 
-  class TestLeoRoutes(leonardoService: LeonardoService)
-                     (override implicit val materializer: Materializer, override val executionContext: ExecutionContext)
-    extends LeoRoutes(leonardoService, SwaggerConfig())(materializer, executionContext)
```

Comment:
It didn't go away, I just removed the unnecessary implicits in the `TestLeoRoutes` constrcutor :)